pkg/storage-init/storage-init.sh

```diff
@@ -154,6 +154,9 @@ for BLK_DEVICE in $BLK_DEVICES; do
     fi
 done
 
+# Create null_blk device
+modprobe null_blk queue_mode=0 gb=1000 bs=4096 irqmode=0 nr_devices=1
```

Comment:
I don't think this need to be here -- if you want this for testing -- you can always do it later on @vk-en